front/src/common/RollingStockSelector/RollingStockSelector.tsx

```diff
@@ -73,7 +67,14 @@ export default function RollingStockSelector({ condensed }) {
                 />
                 <div className="rollingstock-container-img ml-4">
                   <div className="rollingstock-img d-flex align-items-center">
-                    <RollingStock2Img rollingStock={rollingStockSelected} />
+                    {rollingStockSelected.name !== 'XYZ77777' ? (
+                      <RollingStock2Img rollingStock={rollingStockSelected} />
+                    ) : (
+                      <>
+                        <img src={exampleRollingStockImage1} alt="rollingStockHead" />
+                        <img src={exampleRollingStockImage2} alt="rollingStockTail" />
+                      </>
+                    )}
```

Comment:
This is not very clear. Can you create a function above getRollingStockImage like this : 
```
const rollingStockImage = useMemo(() => {
    if (isStoryBook) {
      return (
        <>
          <img src={exampleRollingStockImage1} alt="rollingStockHead" />
          <img src={exampleRollingStockImage2} alt="rollingStockTail" />
        </>
      );
    }
    return rollingStockSelected ? <RollingStock2Img rollingStock={rollingStockSelected} /> : null;
  }, [rollingStockSelected]);
```
and add a prop isStoryBook ? Also, it would be nice to have only 1 image instead of 2. I think you can go  on your local, launch the frontend, open the rollingStock modal and look for the XYZ77777 rollingStock, there you can save the compound image (1 image composed of these 2)

Reply:
done in a different way, let me know what you think about it